osu.Game.Rulesets.Osu/Mods/OsuModVisualAdjusts.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Graphics;
+using osu.Game.Configuration;
+using osu.Game.Rulesets.Mods;
+using osu.Game.Rulesets.Osu.Objects;
+using osu.Game.Rulesets.Osu.UI;
+
+namespace osu.Game.Rulesets.Osu.Mods
+{
+    internal class OsuModVisualAdjusts : ModVisualAdjusts<OsuHitObject, DrawableOsuRuleset>
+    {
+        [SettingSource("Disable follow points", "No more hints for where to follow...")]
+        public DrawableRulesetVisualAdjustSetting DisableFollowPoints { get; } = new DrawableRulesetVisualAdjustSetting(ruleset => ruleset.Playfield.FollowPoints.Hide());
+
+        [SettingSource("No combo colours", "The combo colours won't tell you anything now...")]
+        public DrawableHitObjectVisualAdjustSetting NoComboColours { get; } = new DrawableHitObjectVisualAdjustSetting(dho => dho.AccentColour.Value = Colour4.White);
+    }
```

Comment:
I'm *so confused by this*. There's too much magic. Please write this similar to other mods and do away with the magic.

Reply:
I mean there isn't much magic here, a `VisualAdjustSetting` is just a setting that handles what to do when the setting is active, wouldn't documenting be enough?

Reply:
Just noticed that a simple structure will do better... both optimization wise and for the developer i guess, will make it when i get home

Reply:
I take back on this i will try implementing what i think should be the most understandable when i get home.

Reply:
> I'm *so confused by this*. There's too much magic. Please write this similar to other mods and do away with the magic.

Could you take a look at my latest commit, although the structure follow the same pattern it is better explained now i believe, and it is now handling the `NoComboColours` part correctly.